drivers/include/periph/flashpage.h

```diff
@@ -27,6 +23,20 @@
  *              around 10K times), so using this interface in some kind of loops
  *              can damage you MCU!
  *
+ * @section     sec_pm (Low-) power implications
+ *
+ * The flashpage driver implementation **should** make sure, that the CPU uses
+ * no additional energy while the flashpage driver is active. This means, that
+ * any particular CPU peripherals used for reading and writing flash pages
+ * **should** be disabled before the read and write functions return.
+ *
```

Comment:
This implies power toggling whithin e.g.,

```
char buf[1024];
int src_start = 0, dst_start = 10000; len = 1000;
for (unsigned i = 0; i < 1000; i++) {
    flashpage_read(buf, src_start + i);
    flashpage_write(buf, dst_start + i);
}
```

which might be undesired.

Reply:
I think simplicity wins here.  It is more likely that the larger blocks or at least fewer calls will be used for each power cycle, making the overhead added by hiding the pm control in the functions negligible.
As a worst case someone can implement a pm_keep_awake function if there isn't already one...